GOVERNANCE.md

```diff
@@ -0,0 +1,75 @@
+- Maintainers will make a good-faith effort to monitor the fraction of the meeting time that they speak. The meeting time is limited, and several people may have something interesting to say, report on their updates, or contribute constructive arguments. One should be self-aware and notice if one is dominating conversations, and attempt to make a conscious effort to include other voices and perspectives in the discussion.
+- Maintainers consider it their responsibility to voice their questions and/or concerns as early as possible.  This will ensure that the organization addresses concerns rapidly and effectively.
+- Maintainers consider it their responsibility to listen and understand questions or concerns raised by other partners or through concerns communicated through email to individual *Maintainers*.
+
+
+### Approval by Vote:  
+Many of the decisions in the organization will be enacted after an approval by majority vote of the Maintainers. 
+The process of listening and adjusting strategies to meet everyone's needs is vital for power-sharing organizations.  
+Importantly, spending a little time up front to listen and resolve concerns reduces the pain and time spent later in intense conflict resolution.  
+
+### Communication and Connection: 
+To facilitate open and effective communication, the `Phoebus` will offer multiple levels of communication.  
+Developers will communicate via email, GitHub, and a `Phoebus` Slack channel. 
+
+To respond to user inquiries, the Phoebus Project will host a mailing list via Google or similar.  
+To maintain a standing record of user inquiries and responses, we will post the inquiries and responses on a public-facing site.  
+All Maintainers will have the option to respond to the inquires; to ensure that someone is responsive to the user inquiries, 
+the team will triage tickets/questions in a regular developer call.
+
+Monthly Maintainer meetings will also be an important part of ensuring productivity and timeliness of the meeting.  
+Each meeting will last up to 60 minutes.  We will assess and discuss the goals from the previous meeting.  
+In particular, we will take note which goals were completed and which were not.  If there are uncompleted goals, 
+we will assess why they were not completed.  Next, we will discuss the goals for the upcoming month, quarter, and year.  
+We will decide which goals to move forward during the next month.  
+All of this will be recorded on an online document which will be available to all Maintainers.
+
+Each meeting of the core Maintainers will be managed by a meeting manager.  
+The primary job of the meeting manager is to: 
+1) gather a list of agenda items for the next meeting, 
+2) ensure that the meeting runs on schedule, and 
+3) ensure equitable participation by all participants.   
+Again, in keeping with power-sharing, the meeting manager position will rotate among the core Maintainers, even junior members.
+
```

Comment:
Given the small number of people currently contributing, I think we should add a clause that the meeting cadence (and strictness of above rules) may depend on project activity.

Reply:
I was thinking about this myself when putting it in... I'll add that.